cfn-resources/private-endpoint-adl/README.md

```diff
@@ -0,0 +1,73 @@
+# MongoDB::Atlas::PrivateEndpointADL
+
+## Description
+Adds one private endpoint for Federated Database Instances and Online Archives to the specified projects. To use this resource, the requesting API Key must have the Project Atlas Admin or Project Charts Admin roles. This resource doesn't require the API Key to have an Access List.
+
+## Attributes & Parameters
+
+Please consult the [Resource Docs](docs/README.md)
+
+## Unit Testing Locally
+
+The local tests are integrated with the AWS `sam local` and `cfn invoke` tooling features:
+
+```
+sam local start-lambda --skip-pull-image
+```
+then in another shell:
+```bash
+repo_root=$(git rev-parse --show-toplevel)
+source <(${repo_root}/quickstart-mongodb-atlas/scripts/export-mongocli-config.py)
+cd ${repo_root}/cfn-resources/private-endpoint-adl
+./test/cluster.create-sample-cfn-request.sh YourProjectID YourClusterName > test.request.json 
+echo "Sample request:"
+cat test.request.json
+cfn invoke resource CREATE test.request.json 
+cfn invoke resource DELETE test.request.json 
+cd -
+```
+
+Both CREATE & DELETE tests must pass.
+
+## Installation
+
+Installation currently requires the follow 2 steps to build and then submit/register the
+new MongoDB::Atlas::PrivateEndpointADL Resource Type into your AWS Region. Note, this command uses the
+default AWS region.
+
+```bash
+TAGS=logging make
+cfn submit --verbose --set-default
+```
+## Cloudformation Examples
+
+Please see the [test/inputs_1_create.template.json](test/inputs_1_create.template.json) for example resource for example resource.
```

Comment:
Can you move example to dedicated Examples folder: https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples